src/app.js

```diff
@@ -1,42 +1,84 @@
 const { app } = require("./support/setupExpress");
-const { query } = require("./support/db");
+// const { query } = require("./support/db");
 const { gameOfThronesEpisodes } = require("./data/gameOfThronesData");
-
+const { filterThroughEpisodes } = require("./functions.js");
```

Comment:
line 4-6 can all be stores inside the same object and separated with a comma {fnA, fnB, fnC}